main.tf

```diff
@@ -58,8 +58,9 @@ resource "aws_dynamodb_table" "this" {
     for_each = var.replica_regions
 
     content {
-      region_name = replica.value.region_name
-      kms_key_arn = lookup(replica.value, "kms_key_arn", null)
+      region_name    = replica.value.region_name
+      kms_key_arn    = lookup(replica.value, "kms_key_arn", null)
+      propagate_tags = replica.value.propagate_tags
```

Comment:
As it is an optional, can we default it to false plz by using the lookup function to avoid null pointer errors?